config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
+  # config.default_per_page = 25
```

Comment:
nope, the commented-out configs are the default values - kaminari autogenerates this file